src/test/java/guiTest/ForgotPasswordTest.java

```diff
@@ -0,0 +1,54 @@
+package guiTest;
+
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import pages.ForgotPassword;
+import utils.FunctionalTest;
+
+import static org.junit.Assert.*;
+import static utils.Constants.*;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+
+public class ForgotPasswordTest extends FunctionalTest {
+
+    ForgotPassword forgotPass = new ForgotPassword(driver);
+
+    @Before
+    public void init() {
+        driver.get(FORGOT_PASS_URL);
+    }
+
+    @Test
+    public void test01_noEmail_ResetMyPasswordButtonIsDisabled() {
+        assertFalse(forgotPass.isResetMyPasswordButtonEnabled());
+    }
+
```

Comment:
Should be a part of test04

Reply:
The test01 is verify that the button is disabled with no entered data(email). 
While test04 is verify if the already registered user receives a confirmation message to reset password.